plugins/board/src/index.ts

```diff
@@ -53,9 +53,18 @@ export interface CardLabel extends AttachedDoc {
 export interface Card extends Task {
   title: string
 
+  date?: {
```

Comment:
You should extract it in another interface. And please do same changes in model

Reply:
Extracted `CardDate` into separate interface, but this part `And please do same changes in model` I do not get.

Reply:
Updated